builtins/amp-img.js

```diff
@@ -85,7 +87,11 @@ export function installImg(win) {
       }
       this.img_.setAttribute('src', src);
 
-      return loadPromise(this.img_);
+      return loadPromise(this.img_).catch(error => {
+        if (isProxyOrigin(src)) {
+          reportError(error, this.element);
```

Comment:
`layoutCallback` should fail in the end. So, yes, perhaps just rethrowing a more specific error would be better.